spec.md

```diff
@@ -288,6 +288,27 @@ encapsulated within the `data` attribute.
 * Constraints:
   * OPTIONAL
 
+# Privacy & Security Considerations
+Interoperability is the primary driver behind this specification, enabling such
+behavior requires information to be made available *in the clear* resulting
+in the potential for information leakage.
+
+* Context Attributes
+
+  Sensitive information *SHOULD NOT* be carried or represented in context attributes.
+  
+  CloudEvent participants *MAY* introspect and log context attributes.
+  
+* Data
+
+  Domain specific [data](#data) *SHOULD* be encoded to restrict visibility to
+  trusted parties. 
```

Comment:
We may need to add something like: The exact mechanism by which it is encoded is out of scope of this specification.

Reply:
What does "encoded" mean here?

Reply:
I've changed the wording - please take another look.

@clemensv I was simply trying to avoid the word 'encrypt' as i felt that was to loaded.